queries.sql

```diff
@@ -13,3 +13,41 @@ SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name <> 'Gabumon';
 
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+-- Transaction: Update species column and rollback
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+-- Transaction: Update species based on conditions
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+COMMIT;
```

Comment:
:red_circle: Before you commit the transaction. The [milestone requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md) state that you should verify that changes were made. :arrow_down: . Kindly make sure you include the query for this requirement.

![CHANGE](https://github.com/Eccb7/vet-clinic-database/assets/26736582/64ac800d-6ca4-4cb0-9ae3-04227bb4d51c)

Reply:
Hi @bienvenuushindi 
Thanks for the update.